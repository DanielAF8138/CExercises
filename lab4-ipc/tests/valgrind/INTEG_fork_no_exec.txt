==3308076== Memcheck, a memory error detector
==3308076== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==3308076== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==3308076== Command: ../ipc asldkfja ../data/f1.txt
==3308076== 
==3308121== 
==3308121== HEAP SUMMARY:
==3308121==     in use at exit: 0 bytes in 0 blocks
==3308121==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==3308121== 
==3308121== All heap blocks were freed -- no leaks are possible
==3308121== 
==3308121== For lists of detected and suppressed errors, rerun with: -s
==3308121== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
asldkfja(../data/f1.txt) = ''
==3308076== 
==3308076== HEAP SUMMARY:
==3308076==     in use at exit: 0 bytes in 0 blocks
==3308076==   total heap usage: 3 allocs, 3 frees, 8,293 bytes allocated
==3308076== 
==3308076== All heap blocks were freed -- no leaks are possible
==3308076== 
==3308076== For lists of detected and suppressed errors, rerun with: -s
==3308076== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
