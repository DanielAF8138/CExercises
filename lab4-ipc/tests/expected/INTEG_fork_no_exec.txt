asldkfja(../data/f1.txt) = ''
